src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/IsMatch.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+#pragma warning disable SA1649 // File name should match first type name
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // IsMatch(text:s, format:s)
+    // Checks if the input text is of the correct format.
+    internal sealed class IsMatchFunction : BuiltinFunction
```

Comment:
>IsMatchFunction [](http://example.com/codeflow?start=26&length=15)

are these coming from PA? (so we'll remove the corresponding TexlFunction class in PA and use a single one?)

Reply:
That's the plan Carlos and I have discussed